core/module.h

```diff
@@ -271,6 +279,18 @@ class Module {
 
   const std::vector<bess::OGate *> &ogates() const { return ogates_; }
 
+  int ComputePlacementConstraints() const;
```

Comment:
Method comment.  Also, would it be better to use a uint64_t just to be explicit?